extensions/src/services/http-svc.js

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
I think no need to use -svc prefix In filename as file already lays in services directory